backend/controllers/innovation.go

```diff
@@ -59,3 +65,108 @@ func RequestToInnovationSync(postBody []byte, innovationSlug string) (models.Ser
 
 	return data, nil
 }
+
+func RequestToFaceOcclusionAttribute(ctx *gin.Context, postBody []byte) {
+	// Request to Face Detection API
+	resultFaceDetection, err := RequestToInnovationSync(postBody, "face-detection")
+	if err != nil {
+		ctx.JSON(http.StatusInternalServerError, gin.H{
+			"ok":      false,
+			"message": err.Error(),
+		})
+		return
+	}
+
+	resultFaceDetectionJson, err := json.Marshal(resultFaceDetection)
+	if err != nil {
+		log.WithFields(log.Fields{
+			"error":     	err,
+			"json_result": 	resultFaceDetectionJson,
```

Comment:
ini `byte` kan? ke print di log nya gimana? perlu di `string(resultFaceDetectionJson)` gak? klo udh readable gpp

Reply:
@krismp oh iya itu `byte`, sorry kelewat. Nice catch, thank you, Mas.

Reply:
done https://github.com/nodefluxio/weber/pull/177/commits/703bc50d18b7798412092919cf5b9b2c51fbaa97